core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -145,31 +149,46 @@ var PreferencesPage = function() {
   // might be abbreviated), rather than the text on the popover that appears
   // when hovering over the tile.
   this.expectDisplayedFirstSubscriptionToBe = async function(name) {
+    await waitFor.visibilityOf(
+      subscriptions, 'subscriptions taking too long to appear.');
```

Comment:
Please wait for `subscriptions.first()` instead of `subscriptions`. `subscriptions` refers to a bunch of elements, but protractor's wait functions expect a single element. See the documentation [here](https://www.protractortest.org/#/api?view=ProtractorExpectedConditions.prototype.visibilityOf).